frontend/tests/controllers/ProblemUpdateTest.php

```diff
@@ -71,7 +71,7 @@ public function testUpdateProblemLevel() {
         $problemLevel = \OmegaUp\DAO\ProblemsTags::getProblemLevel(
             $problemData['problem']
         );
-        $this->assertNull($problemLevel);
+        //$this->assertNull($problemLevel);
```

Comment:
```suggestion
        $this->assertEqual($problemLevel, ...);
```

en general, el código comentado jamás se debe subir a producción. si se desea eliminar algo, es mejor eliminarlo de verdad. así es más fácil identificar la intención del autor. _pero_ en este caso podemos aseverar que _no_ es nulo!